sharktuner/sharktuner/candidate_ordering.py

```diff
@@ -4,6 +4,7 @@
 import logging
 
 from . import common
+from iree.compiler.dialects import iree_gpu  # type: ignore
```

Comment:
```suggestion
import logging

from iree.compiler.dialects import iree_gpu  # type: ignore

from . import common
```

1. Standard library (logging)
2. Third-party libraries (iree)  
3. Local imports (. import common)